### DISCUSSION TITLE
Newly created accounts

### DISCUSSION THREAD

**Discussion by ghost** [efroemling/ballistica#298] [Ideas]
Is there any way to get the user **registration date** (by account_id)?
It will be very useful against spammers (they take places in the lobby and don't allow to enter to the server)

**Comment by efroemling** [comment]
Ok, I just added a 'created' entry to the json results returned by account-query.
This gives you a year, month, day, hour, minute, and second in UTC time.
Example: http://bombsquadgame.com/accountquery?id=GC-G:9393647

Please let me know if there is other info that would be handy to have returned here..
